app/views/restrooms/_formsubmit.html.haml

```diff
@@ -7,6 +7,9 @@
       = t('restroom.guess_location')
       %i.fa.fa-refresh.fa-spin
 
+  / Add map for preview
+  %div#mapArea.hidden{ data: { latitude: 38.904735, longitude: -77.033812 } }
```

Comment:
Super minor thing, you don't need the `%div`, you can just start with the id and haml defaults to making it a div.